src/components/fields/__tests__/DataSelector-test.js

```diff
@@ -38,7 +38,7 @@ describe('DataSelector', () => {
     const onUpdateTraces = jest.fn();
     const wrapper = render({onUpdateTraces}).find(DropdownWidget);
     wrapper.prop('onChange')('y1');
-    expect(onUpdateTraces.mock.calls[0][0]).toEqual({
+    expect(onUpdateTraces.mock.calls[2][0]).toEqual({
```

Comment:
You could clear the mock after the render right before calling `onChange`. That way we are impervious to these changes in indexes.